src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java

```diff
@@ -137,6 +137,25 @@
  * }
  * </pre>
  * <p>
+ * An example of the default configuration for all methods:
+ * </p>
+ * <pre>
+ * &lt;module name="MethodName"&gt;
+ *    &lt;property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   public void first_Method1() {} // OK
+ *   protected void second_Method2() {} // OK
+ *   private void Third_Method3() {} // violation, name 'ThirdMethod' must match
+ *                                 // pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
+ *   void Fourth_Method4() {} // violation, name 'FourthMethod' must match
+ *                          // pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
+ * }
+ * </pre>
+ * <p>
```

Comment:
If its an example of default config then write it below Line no. 85 in this file, since there you have to add code examples of default config.

Reply:
Resolved